astropy/modeling/tests/test_models.py

```diff
@@ -53,12 +52,13 @@ def sine_deriv(x, amplitude=4, frequency=1):
     sin_model.evaluate(x, 5., 2.)
     sin_model.fit_deriv(x, 5., 2.)
 
-    np.random.seed(0)
-    data = sin_model(x) + np.random.rand(len(x)) - 0.5
+    rng = np.random.default_rng(42)
+    data = sin_model(x) + rng.uniform(size=len(x)) - 0.5
     fitter = fitting.LevMarLSQFitter()
     model = fitter(sin_model, x, data)
-    assert np.all((np.array([model.amplitude.value, model.frequency.value]) -
-                   np.array([amplitude, frequency])) < 0.001)
```

Comment:
@nden: This was only passing with 0.001 because some of the differenced values were < 0 (it should be wrapped in an `np.abs()`). So the new tolerance here I think is also consistent with past values, once accounting for the abs.